tensornetwork/backends/numpy/decompositions_test.py

```diff
@@ -52,6 +52,13 @@ def test_qr(self):
       q, r = decompositions.qr(np, random_matrix, 1, non_negative_diagonal)
       self.assertAllClose(q.dot(r), random_matrix)
 
+  def test_cholesky(self):
+    #Assured positive-definite hermitian matrixs
+    random_matrix = [[1.0, 0], [0, 1.0]]
```

Comment:
Using the identity matrix  as "random" matrix is not an ideal test case. You can generate a positive definite matrix e.g. by
```python 
A = np.random.rand(D,D)
A  = A @ A.T.conj()
```

Reply:
btw. also test this for all supported dtypes

Reply:
do `np.random.rand(D, D).astype(dtype)` to get your fav type. if it's imaginary you might need to initialize the real and imaginary parts separately.

put `np.random_seed(10)` (or any other fixed integer) before the `rand` so that the matrix is fixed.

Reply:
There is no `dtype` defined in any of the tests, should I define one myself?